docs/docs/1.2.3/policies/mutual-tls.md

```diff
@@ -187,6 +187,48 @@ A few considerations:
 * The `dpCert` configuration determines how often Kuma should automatically rotate the certificates assigned to every data plane proxy.
 * The Secrets must exist before referencing them in a `provided` backend.
 
+## Permissive mTLS
+
+Kuma provides a convenient way to migrate existing workloads to the mTLS mesh with zero downtime. In order to do so 
+`PERMISSIVE` mode has to be enabled. 
```

Comment:
```suggestion
In version 1.3.0 and later, Kuma provides `PERMISSIVE` mTLS mode to let you migrate existing workloads with zero downtime:
```

This ... seems like something we should maybe also encourage users not to leave on? Or at the very least warn about insecure incoming connections? Or ... something?

I'm also not sure about the order of things. Might be better to put the explanation at lines 228-230 before the examples also?

Reply:
Good point. I'm missing the information on when can I switch from permissive to strict mTLS. Metrics are in progress?

Reply:
Added a warning: 

![image](https://user-images.githubusercontent.com/12110261/130803052-96937f56-2287-435b-bbd1-af7373e5c8fd.png)

The message and format is not final, I know @Bradamant3 doesn't like a lot of frames :)